graylog2-plugin-interfaces/src/test/java/org/graylog2/plugin/BaseConfigurationTest.java

```diff
@@ -44,6 +44,9 @@
         @Parameter(value = "rest_listen_uri", required = true)
         private URI restListenUri = URI.create("http://127.0.0.1:12900/");
 
+        @Parameter(value = "web_listen_uri", required = true)
+        private URI webListenUri = URI.create("http://127.0.0.1:9000/");
```

Comment:
Follow up task: I would like to proxy the REST API on port 9000 as well, so that deployments which have restricted Graylog traffic to port 9000 continue to work. This is out of scope for this branch, but for reference we should be able to use https://jersey.java.net/documentation/latest/filters-and-interceptors.html Section 10.2.1.1. Pre-matching and post-matching filters